src/test/java/org/apache/sling/graphql/core/mocks/URLScalarConverter.java

```diff
@@ -41,4 +41,14 @@
         final String testPrefix = getClass().getSimpleName() + " says:";
         return testPrefix + (value == null ? null : value.toExternalForm());
     }
+
+    @Override
+    public String getName() {
```

Comment:
we have name and description in `SlingScalarConverter` here it's not yet implemented